<launch>
        <!-- joy node -->
        <node respawn="true" pkg="joy" type="joy_node" name="BobCore" >
                <param name="dev" type="string" value="/dev/input/js0" />
                <param name="deadzone" value="0.12" />
                <param name="default_trig_val" value="true" />
        </node>

        <!-- Axes -->
        <param name="axis_linear" value="1" type="int" />
        <param name="axis_angular" value="0" type="int" />
        <param name="scale_linear" value="1" type="double" />
        <param name="scale_angular" value="1" type="double" />

        <!-- Buttons -->
        <param name="b_button" value="1" type="int" />
        <param name="y_button" value="1" type="int" />

        <node pkg="bob2" type="XBoxController" name="xbox" respawn="true" respawn_delay="3"/>
        <node pkg="bob2" type="RunMotor" name="runMotor" output="screen" />
        <node pkg="bob2" type="qr_code_detection.py" name="qr_code" />
        <node pkg="bob2" type="Gyroscope" name="gyro" />
        <node pkg="bob2" type="Motor_Speed_Node" name="motor_speed" />

        <node name="rplidarNode"          pkg="rplidar_ros"  type="rplidarNode" output="screen">
        <param name="serial_port"         type="string" value="/dev/ttyUSB0"/>
        <param name="serial_baudrate"     type="int"    value="115200"/>
        <param name="frame_id"            type="string" value="laser"/>
        <param name="inverted"            type="bool"   value="false"/>
        <param name="angle_compensate"    type="bool"   value="true"/>
        </node>

</launch>
